packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -116,11 +117,25 @@ interface Relation extends Pick<RelationResult, 'documentId' | 'id' | 'locale' |
   label: string;
   position?: RelationPosition;
   __temp_key__: string;
+  apiData?: {
+    documentId: RelationResult['documentId'];
+    id: RelationResult['id'];
+    locale?: RelationResult['locale'];
+    position: RelationPosition;
+  };
```

Comment:
Why is this added?

Reply:
we need the apiData in this handler handleChangeModalContent https://github.com/strapi/strapi/blob/18663acd7bf8a6b72833694b0257609103a7a4c5/packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx#L1472